### DISCUSSION TITLE
RabbitMq message broker

### DISCUSSION THREAD

**Discussion by ujjawaldyson** [StackStorm/st2#6179] [Operations]
i want to write my rule criteria that action should be triggered when short description contains "xyz"

[root@Stackstorm rules]# st2 trigger-instance get 660fc7200d18301e3b123f98
+-----------------+--------------------------------------------------------------+
| Property        | Value                                                        |
+-----------------+--------------------------------------------------------------+
| id              | 660fc7200d18301e3b123f98                                     |
| trigger         | rabbitmq.new_message                                         |
| occurrence_time | 2024-04-05T09:40:48.373000Z                                  |
| payload         | {                                                            |
|                 |     "body": "{"incident_number": "INC0010491", "opened_at":  |
|                 | "2024-04-05 09:40:48", "priority": "5", "short_description": |
|                 | "This incident Automation", "assignment_group": null,        |
|                 | "number": "INC0010491", "sys_id":                            |
|                 | "f28072b2939102109ee77bfdfaba1046"}",                        |
|                 |     "queue": "servicenow_incidents"                          |
|                 | }                                                            |
| status          | processed                                                    |
+-----------------+--------------------------------------------------------------+

Can anybody please help me to write the correct criteria

**Comment by fdrab** [comment]
have you tried trigger.body.short_description contains xyz?
